bin/reth/src/node/mod.rs

```diff
@@ -98,10 +98,11 @@ impl Command {
         let consensus = Arc::new(BeaconConsensus::new(self.chain.consensus.clone()));
         let genesis_hash = init_genesis(db.clone(), self.chain.genesis.clone())?;
 
-        info!("Connecting to p2p");
         let network =
             config.network_config(db.clone(), chain_id, genesis_hash).start_network().await?;
 
+        info!(peer_id = %network.peer_id(), local_addr = %network.local_addr(), "Started p2p networking");
```

Comment:
the display impl of PeerId truncates, do we want to log the full PeerId?

Reply:
Ah yeah true, changing